rfc-035-use-standard-linting-tools-across-all-our-projects.md

```diff
@@ -0,0 +1,37 @@
+# Use standard linting tools across all of our projects
+
+## Summary
+
+Where available, all new dxw-owned or maintained repositories SHOULD use a
```

Comment:
Have some words been snipped from here? This sentence doesn't parse for me.

Reply:
Whoops - good catch!